core-java-modules/core-java-streams-4/README.md

```diff
@@ -6,3 +6,4 @@
 - [Java 8 Streams: Multiple Filters vs. Complex Condition](https://www.baeldung.com/java-streams-multiple-filters-vs-condition)
 - [Finding Max Date in List Using Streams](https://www.baeldung.com/java-max-date-list-streams)
 - [Batch Processing of Stream Data in Java](https://www.baeldung.com/java-stream-batch-processing)
+- [Converting String arrays to Integers using Streams](https://www.baeldung.com/mapping-an-array-of-integers-to-strings-using-java-streams/)
```

Comment:
hm, the link doesn't work: https://www.baeldung.com/mapping-an-array-of-integers-to-strings-using-java-streams/

Reply:
Thanks for the comment, I wasn't sure what to do here as the article isn't published yet. Should I remove the README addition from here until the article is live?